docker/build-tools/Dockerfile

```diff
@@ -77,45 +77,46 @@ RUN unzip /tmp/protoc-${PROTOC_VERSION}-linux-x86_64.zip
+RUN GO111MODULE=on go get -ldflags="-s -w" github.com/gogo/protobuf/protoc-gen-gogoslick@${GOGO_PROTOBUF_VERSION}
+RUN GO111MODULE=on go get -ldflags="-s -w" github.com/uber/prototool/cmd/prototool@${PROTOTOOL_VERSION}
+RUN GO111MODULE=on go get -ldflags="-s -w" github.com/nilslice/protolock/cmd/protolock@${PROTOLOCK_VERSION}
+RUN GO111MODULE=on go get -ldflags="-s -w" github.com/maxbrunsfeld/counterfeiter/v6@${COUNTERFEITER_VERSION}
+RUN GO111MODULE=on go get -ldflags="-s -w" golang.org/x/tools/cmd/goimports@${GOIMPORTS_VERSION}
+RUN GO111MODULE=on go get -ldflags="-s -w" github.com/golangci/golangci-lint/cmd/golangci-lint@${GOLANGCI_LINT_VERSION}
+RUN GO111MODULE=on go get -ldflags="-s -w" github.com/jteeuwen/go-bindata/go-bindata@${GO_BINDATA_VERSION}
+RUN GO111MODULE=on go get -ldflags="-s -w" github.com/envoyproxy/protoc-gen-validate@${PROTOC_GEN_VALIDATE_VERSION}
+RUN GO111MODULE=on go get -ldflags="-s -w" github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway@${PROTOC_GEN_GRPC_GATEWAY_VERSION}
+RUN GO111MODULE=on go get -ldflags="-s -w" github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger@${PROTOC_GEN_SWAGGER_VERSION}
+RUN GO111MODULE=on go get -ldflags="-s -w" github.com/jstemmer/go-junit-report@${GO_JUNIT_REPORT_VERSION}
+RUN GO111MODULE=on go get -ldflags="-s -w" sigs.k8s.io/kind@${KIND_VERSION}
+RUN GO111MODULE=on go get -ldflags="-s -w" github.com/wadey/gocovmerge@${GOCOVMERGE_VERSION}
+RUN GO111MODULE=on go get -ldflags="-s -w" github.com/fortio/fortio@${FORTIO_VERSION}
+RUN GO111MODULE=on go get -ldflags="-s -w" github.com/imsky/junit-merger/src/junit-merger@${JUNIT_MERGER_VERSION}
 
 # Install latest version of Istio-owned tools
-RUN GO111MODULE=on go get istio.io/tools/cmd/protoc-gen-docs
-RUN GO111MODULE=on go get istio.io/tools/cmd/annotations_prep
-RUN GO111MODULE=on go get istio.io/tools/cmd/cue-gen
-RUN GO111MODULE=on go get istio.io/tools/cmd/envvarlinter
-RUN GO111MODULE=on go get istio.io/tools/cmd/testlinter
-RUN GO111MODULE=on go get istio.io/tools/cmd/protoc-gen-deepcopy
-RUN GO111MODULE=on go get istio.io/tools/cmd/protoc-gen-jsonshim
-RUN GO111MODULE=on go get istio.io/tools/cmd/kubetype-gen
-RUN go get -u istio.io/test-infra/toolbox/githubctl
-RUN go get -u istio.io/test-infra/boskos/cmd/mason_client
+RUN GO111MODULE=on go get -ldflags="-s -w" istio.io/tools/cmd/protoc-gen-docs
+RUN GO111MODULE=on go get -ldflags="-s -w" istio.io/tools/cmd/annotations_prep
+RUN GO111MODULE=on go get -ldflags="-s -w" istio.io/tools/cmd/cue-gen
+RUN GO111MODULE=on go get -ldflags="-s -w" istio.io/tools/cmd/envvarlinter
+RUN GO111MODULE=on go get -ldflags="-s -w" istio.io/tools/cmd/testlinter
+RUN GO111MODULE=on go get -ldflags="-s -w" istio.io/tools/cmd/protoc-gen-deepcopy
+RUN GO111MODULE=on go get -ldflags="-s -w" istio.io/tools/cmd/protoc-gen-jsonshim
+RUN GO111MODULE=on go get -ldflags="-s -w" istio.io/tools/cmd/kubetype-gen
+RUN GO111MODULE=on go get -ldflags="-s -w" istio.io/tools/cmd/check_licenses
```

Comment:
not sure how fast GOPROXY is, will we hit a race where we don't pick up the changes? I guess that only matters if we edit the tool and docker in a single commit which is rare